packages/renderer/src/Route.svelte

```diff
@@ -13,10 +13,14 @@ export let redirect = false;
 export let firstmatch = false;
 export let breadcrumb: string | undefined = undefined;
 export let navigationHint: NavigationHint | undefined = undefined;
+export let requestParser:
+  | ((request: { query: Record<string, string>; params: Record<string, string> }) => Request)
+  | undefined = undefined;
 
 let showContent = false;
 let params: Record<string, string> = {};
 let meta: TinroRouteMeta = {} as TinroRouteMeta;
+let request: Request = {} as Request;
```

Comment:
question: could we also use `Request | undefined` to avoid passing an empty object with a cast?

Reply:
I changed to undefined, thanks